src/main/python/ttconv/tt.py

```diff
@@ -39,6 +39,10 @@
 
 LOGGER = logging.getLogger(__name__)
 
```

Comment:
Writing out the "." should probably be throttled by time or something else such that we don't fill up our terminal

Reply:
What about the callback returning a value between 0 and 1?

Reply:
Done